src/components/ComboButton/ComboButton.stories.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @file Combo button stories.
+ * @copyright IBM Security 2019
+ */
+
+import React from 'react';
+import centered from '@storybook/addon-centered/react';
+import { storiesOf } from '@storybook/react';
+import { withA11y } from '@storybook/addon-a11y';
+import { radios } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import ArrowRight20 from '@carbon/icons-react/lib/arrow--right/20';
+import Filter20 from '@carbon/icons-react/lib/filter/20';
+
+import { patterns } from '../../../.storybook';
+
+import { TooltipDirection } from '../IconButton/IconButton';
+import { ComboButton, ComboButtonItem } from '../..';
+
+ComboButtonItem.displayName = 'ComboButtonItem';
+
+const props = () => ({
+  direction: radios(
+    'Menu direction (direction)',
+    {
+      top: TooltipDirection.TOP,
+      bottom: TooltipDirection.BOTTOM,
+    },
+    TooltipDirection.TOP
+  ),
+});
+
+storiesOf(patterns('ComboButton'), module)
+  .addDecorator(withA11y)
+  .addDecorator(centered)
+  .add(
+    'default',
+    () => (
+      <div style={{ width: '300px' }}>
+        <ComboButton {...props()}>
+          <ComboButtonItem
+            onClick={action('onClick (item 1)')}
+            renderIcon={ArrowRight20}
+          >
+            <span title="Item 1 (becomes primary button and text will be truncated)">
+              Item 1 (becomes primary button and text will be truncated)
+            </span>
+          </ComboButtonItem>
+          {Array(5)
+            .fill(0)
+            .map((item, index) => {
+              const text = `Item ${index +
+                1} - text may be long and will be truncated`;
+              return (
+                <ComboButtonItem
+                  className="some-class"
+                  key={`combo-button-item-'${item.id}`}
+                  index={index}
+                  onClick={action(`onClick ("${text}")`)}
+                >
+                  <span title={text}>{text}</span>
+                  <Filter20 />
```

Comment:
I would make the API for the `ComboButtonItem` consistent. I like the use of the `renderIcon` that is used for the primary action instead of having the user provide their own children markup for the overflow actions.